src/tests/validation/validate-new-user.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import express from 'express';
+import db from '../../models';
+import validateNewUser from '../../middlewares/validate-new-user';
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+const app = express();
+const router = express.Router();
+
+app.use(express.json());
+app.use(
+  express.urlencoded({
+    extended: false
+  })
+);
+
+app.use('/api/users', router.post('/', validateNewUser));
+
+const newUser = {
```

Comment:
there are factory libraries that can be used to generate such data. look at `rosie` and `chance`

Reply:
ok, I'll check this out